identifiers/country-in/lok_sabha.csv

```diff
@@ -154,9 +154,11 @@ ocd-division/country:in/state:jh/cd:rajmahal,Jharkhand Lok Sabha constituency Ra
 ocd-division/country:in/state:jh/cd:ranchi,Jharkhand Lok Sabha constituency Ranchi,,
 ocd-division/country:in/state:jh/cd:singhbhum,Jharkhand Lok Sabha constituency Singhbhum (ST),,
 ocd-division/country:in/state:jk/cd:anantnag,Jammu and Kashmir Lok Sabha constituency Anantnag,,2019-10-30
+ocd-division/country:in/state:jk/cd:anantnag-rajouri,Jammu and Kashmir Lok Sabha constituency anantnag-rajouri,,

```

Comment:
Jammu and Kashmir state no longer exists. Update the constituencies attached to the "Jammu and Kashmir" territory if a renaming occurred.

Reply:
We've updated the request, please refer the updated one.